test/rest_request_template/snowpipe_streaming_string_json_dlq.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Could we have one for schematization as well? Basically all schema difference error shouldn't move rows to DLQ

Reply:
Sounds good, can add that. 
Just clarifying, it shouldn't move to DLQ? 
mind sharing an example?

Reply:
Added DLQ for schema mapping.